.env.schema

```
@@ -0,0 +1,62 @@
+# === AUTH-SYSTEM SCHEMA CONFIGURATION ===
+# Generated by init-auth-schema.js on 2025-05-28T14:16:35.851Z
+# 
+# These credentials connect your app to its dedicated database schema
+# in the Auth-System. Keep these values secure and persistent.
+
+# === CLIENT CREDENTIALS ===
+CLIENT_ID=client_fbbac14b71304afdb591a03ae3133e50
+CLIENT_SECRET=8d499a99-33f2-4731-b57a-941aeebccbaf
```

RuboCop (1.75.5)

[fatal] unexpected token tIDENTIFIER
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)